develop_docker/install-prereqs-and-start-server.bash

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I think this is superseded by `run-dev-env.bash`

Reply:
still up for deletion?

Reply:
removed